src/files.js

```diff
@@ -117,6 +117,7 @@ FOAM_FILES([
   { name: "foam/dao/TimingDAO" },
   { name: "foam/dao/LoggingDAO" },
   { name: "foam/dao/IDBDAO", flags: ['web'] },
+  { name: "foam/dao/SQL" },
```

Comment:
Can we put this behind a 'sql' flag, since it isn't useful as a core library function, but only to those who are also loading some kind of SQL DAO.